rllib/env/multi_agent_episode.py

```diff
@@ -0,0 +1,535 @@
+import uuid
+
+from typing import Any, Dict, List, Optional, Union
+
+from ray.rllib.env.single_agent_episode import SingleAgentEpisode
+from ray.rllib.policy.sample_batch import MultiAgentBatch
+
+
+class IndexMapping(list):
```

Comment:
Nit: Can we move this class to the bottom of the file? I know we still do this a lot in RLlib, but we should always have the main class come first in its file and any helper classes moved to the bottom.

Reply:
Also, we should make this a private class (simply rename to `_IndexMapping`).

Reply:
No problem! Consider it done.